test/v1/controllers/RetrieveItsaStatusControllerSpec.scala

```diff
@@ -17,42 +17,34 @@
 package v1.controllers
 
 import api.controllers.{ControllerBaseSpec, ControllerTestRunner}
-import api.hateoas.HateoasLinks
-import api.mocks.hateoas.MockHateoasFactory
-import api.models.domain.{Nino, TaxYear}
-import api.models.errors.{ErrorWrapper, FutureYearsFormatError, NinoFormatError}
+import api.models.domain.TaxYear
+import api.models.errors.{ErrorWrapper, NinoFormatError}
 import api.models.outcomes.ResponseWrapper
-import play.api.libs.json.{JsValue, Json}
+import config.MockAppConfig
+import play.api.libs.json.Json
 import play.api.mvc.Result
-import v1.mocks.services.MockRetrieveItsaStatusService
-import v1.mocks.validators.MockRetrieveItsaStatusValidator
+import v1.controllers.validators.MockRetrieveItsaStatusValidatorFactory
 import v1.models.domain.{StatusEnum, StatusReasonEnum}
-import v1.models.request.{RetrieveItsaStatusRawData, RetrieveItsaStatusRequest}
+import v1.models.errors.FutureYearsFormatError
+import v1.models.request.RetrieveItsaStatusRequestData
 import v1.models.response.{ItsaStatusDetails, ItsaStatuses, RetrieveItsaStatusResponse}
+import v1.services.MockRetrieveItsaStatusService
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class RetrieveItsaStatusControllerSpec
-  extends ControllerBaseSpec
+    extends ControllerBaseSpec
     with ControllerTestRunner
     with MockRetrieveItsaStatusService
-    with MockHateoasFactory
-    with MockRetrieveItsaStatusValidator
-    with HateoasLinks {
+    with MockRetrieveItsaStatusValidatorFactory
+    with MockAppConfig {
```

Comment:
should be able to remove with once all the deprecation header related logic has been removed